app/views/groups/workflow_executions/show.html.erb

```diff
@@ -83,62 +83,73 @@
   <% end %>
 <% end %>
 
-<%= render Pathogen::TabsPanel.new(id: "workflow-executions-tabs") do |tabs| %>
+<%= render Pathogen::Tabs.new(
+  id: "workflow-executions-tabs",
+  label: t(:"workflow_executions.show.tabs.label"),
+  default_index: @tab_index,
+  sync_url: true
+) do |tabs| %>
   <% tabs.with_tab(
     id: "summary-tab",
-    text: t(:"workflow_executions.show.tabs.summary"),
-    href:
-      group_workflow_execution_path(
-        @namespace,
-        @workflow_execution,
-        tab: "summary",
-      ),
-    selected: @tab == "summary" || @tab == nil,
+    label: t(:"workflow_executions.show.tabs.summary"),
+    selected: @tab == "summary" || @tab == nil
```

Comment:
Suggestion (blocking): now that the controller sets the default you don't need the nil check

Reply:
Removed in 63b80461e30476d91dc378d85b968f46472ccd6b